tasks/main.yml

```diff
@@ -74,6 +133,19 @@
   notify:
     - restart keepalived
 
+- name: Configure keepalived service if snapd
```

Comment:
Can you move that up in the snap section?

Reply:
https://github.com/evrardjp/ansible-keepalived/pull/161/commits/11378045cfc1e52b54371927f5458d181ffb487b should do it

Reply:
Ok for me!